src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -305,19 +305,6 @@ Since internal integrations are automatically installed (and uninstallation is e
 
 The Sentry Integration Platform provides the ability to add rich UI components to Sentry itself through a declarative syntax that requires zero code.
 
-Through a JSON-Schema based system, you can have Sentry render a way for Users to link Sentry Issues to Issues in your service or open a specific line of a stack trace in your tool. We'll be expanding the scope of what you can augment over time, as well.
```

Comment:
This is repeated on the new index page, I wanted to keep this section very brief and mainly just link to the new pages.